lib/Config.js

```diff
@@ -372,7 +372,7 @@ class Config extends EventEmitter {
                     item.substr(0, lastColon);
                 // the port should not include the colon
                 const port = item.substr(lastColon + 1);
-                assert(parseInt(port, 10),
+                assert(Number.parseInt(port, 10),
```

Comment:
should work with or without the `Number.`

Reply:
In 8.0/8.1 they're using `Number.parseInt`, I would prefer to keep this way to avoid additional merge conflicts :')